src/tensor_ops/mod.rs

```diff
@@ -79,3 +79,8 @@ mod reshape;
 
 #[cfg(feature = "nightly")]
 pub use reshape::*;
+
+#[cfg(feature = "nightly")]
+mod impl_reshape;
+#[cfg(feature = "nightly")]
+pub use impl_reshape::*;
```

Comment:
Want to just replace the contents of the reshape file? You weren't able to use the reshape function in there right?

Reply:
I'll get rid of the reshape file. I feel like impl_reshape stays consistent with all the other impl files since it's a trait now, rather than a generic function.